controllers/cron_rollout_helpers.go

```diff
@@ -120,13 +126,29 @@ func createBaseDisruption(owner metav1.Object, disruptionSpec *chaosv1beta1.Disr
 }
 
 // setDisruptionAnnotations updates the annotations of a given Disruption object with those of its owner.
-// Additionally, it sets a scheduled time annotation using the provided scheduledTime.
-func setDisruptionAnnotations(disruption *chaosv1beta1.Disruption, owner metav1.Object, scheduledTime time.Time) {
-	for k, v := range owner.GetAnnotations() {
+// It sets a scheduled time annotation using the provided scheduledTime.
+// It parses the UserInfo annotation if it exists and sets user-related annotations.
+func setDisruptionAnnotations(disruption *chaosv1beta1.Disruption, owner metav1.Object, scheduledTime time.Time, log *zap.SugaredLogger) {
+	// Copy owner annotations to Disruption
```

Comment:
could you just initialise disruptions.Annotations if it's nil?

Reply:
I think it should ever be `nil`, `setDisruptionAnnotations` get called here 
https://github.com/DataDog/chaos-controller/blob/78db1ea3033051c796a0ec884a6bdd450d627d8d/controllers/cron_rollout_helpers.go#L165

after we [initialize base disruption](https://github.com/DataDog/chaos-controller/blob/78db1ea3033051c796a0ec884a6bdd450d627d8d/controllers/cron_rollout_helpers.go#L160) with annotations:
https://github.com/DataDog/chaos-controller/blob/78db1ea3033051c796a0ec884a6bdd450d627d8d/controllers/cron_rollout_helpers.go#L108-L120